doc/api/Playback_Information/getLastStoredContentPosition.md

```diff
@@ -0,0 +1,35 @@
+# getLastStoredContentPosition
+
+## Description
+
+Returns the last stored position of the last played content, in seconds.
+Returns `undefined` if no content was previously loaded.
+
+This method can be useful if you want to reload using `player.loadVideo()` and not
+`player.reload()`, after an error for example.
+It can also be used to store the position of an error.
```

Comment:
Minor nitpick but it is not used "to store the position", when it is used it is to obtain that position [roughly] at the time at which the error arised.

I get what you wanted to write, but the API documentation is in the end mostly for application developer who only care about when they might want to use the API, so the wording that is chosen has to reflect that.